### PR TITLE
use filtering

### PR DESCRIPTION
**Pull request by khous**

### CODE REVIEW

File: cli/hdb [hunk 1]
```diff
@@ -86,7 +86,7 @@ def write_script(source, fname):
 # add scripta as usefulname
 def add_script(args):
     print('retrieving ' + args[0])
-    res = requests.get('https://env-1.test.infoblox.com/hack-db/v1/tool', headers={'Authorization': 'Token token=b4b2c7ea0d724cc3ac38686aac86a6b8'})
+    res = requests.get('https://env-1.test.infoblox.com/hack-db/v1/tool', headers={'Authorization': 'Token token='})
```
Comment (brocwoodworthIBLX) [inline cli/hdb:89]: maybe put a string here that says something like `<USER FILL THIS IN>`? And/or a variable that can be set with a message in the help file

**Review by wsp93 — COMMENTED** [review]
Umm......... Is it just me, or does this look like it still lists all and then uses python code to search through the list? 😅

**Comment by brocwoodworthIBLX** [comment]
> Umm......... Is it just me, or does this look like it still lists all and then uses python code to search through the list? 😅

oh haha, i think you're right

File: cli/hdb (hunk 1 above)
Comment (khous) [reply cli/hdb]: doneski

**Comment by khous** [comment]
> > Umm......... Is it just me, or does this look like it still lists all and then uses python code to search through the list? 😅
> 
> oh haha, i think you're right

It most definitely does. I'm practicing my fast and shitty coding

**Comment by wsp93** [comment]
> > > Umm......... Is it just me, or does this look like it still lists all and then uses python code to search through the list? 😅
> > 
> > 
> > oh haha, i think you're right
> 
> It most definitely does. I'm practicing my fast and shitty coding

😂

**Pull request merged**